lib/server/routes/frames.js

```diff
@@ -351,11 +351,35 @@ FramesRouter.prototype.getFrameById = function(req, res, next) {
  */
 FramesRouter.prototype._definitions = function() {
   return [
-    ['POST', '/frames', this._limiter(defaults), this._verify, this.createFrame],
-    ['PUT', '/frames/:frame', this._limiter(defaults), this._verify, this.addShardToFrame],
-    ['DELETE', '/frames/:frame', this._limiter(defaults), this._verify, this.destroyFrameById],
-    ['GET', '/frames', this._limiter(defaults), this._verify, this.getFrames],
-    ['GET', '/frames/:frame', this._limiter(defaults), this._verify, this.getFrameById]
+    ['POST',
+     '/frames',
+     this.getLimiter(limiter(this._defaults.shardsPerMinute)),
+     this._verify,
+     this.createFrame
+   ],
+   ['PUT',
+     '/frames/:frame',
+     this.getLimiter(limiter(this._defaults.shardsPerMinute)),
```

Comment:
This is the only one that should have `shardsPerMinute`, the others can use a default value for new..